examples/conformer_energies/conformer_energies.ipynb

```diff
@@ -17,60 +31,111 @@
     "from openff.interchange.drivers import get_gromacs_energies, get_openmm_energies"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Choose the molecule we want by specifying a SMILES:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "SMILES = 10 * \"C\"  #  \"c1n(CCO)c(C(F)(F)(F))cc1CNCCl\""
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Generate conformers with the OpenFF Toolkit. Note that these conformers in general are not energy minimums of the force field, any particular QM method, or reality; their purpose is to generate diverse starting points for further study."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "molecule = Molecule.from_smiles(SMILES)\n",
     "molecule.generate_conformers(n_conformers=20, rms_cutoff=0.1 * unit.angstrom),\n",
     "topology = molecule.to_topology()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load the Sage 2.1.0 force field:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
     "sage = ForceField(\"openff-2.0.0.offxml\")"
```

Comment:
```suggestion
    "sage = ForceField(\"openff-2.1.0.offxml\")"
```
:grin:

Reply:
I got half way there with the description :rofl: